nextflow.config

```diff
@@ -280,32 +342,38 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Set default registry for Docker and Podman independent of -profile
+// Will not be used unless Docker / Podman are enabled
+// Set to your registry if you have a mirror of containers
+docker.registry = 'quay.io'
+podman.registry = 'quay.io'
+
```

Comment:
```suggestion
```

Duplication, you can remove

Reply:
Fixed in https://github.com/nf-core/taxprofiler/pull/370